bower.json

```diff
@@ -19,14 +19,14 @@
     "paper-elements": "polymerelements/paper-elements#^1.0.7",
     "carbon-elements": "polymerelements/carbon-elements#^0.1.0",
     "iron-elements": "polymerelements/iron-elements#^1.0.10",
-    "carbon-route": "TimvdLippe/carbon-route#subroute-updating",
+    "carbon-route": "TimvdLippe/carbon-route#reset-tail",
```

Comment:
Why is carbonroute still in your fork, when it has been updated?
Or was it only partly merged?